backend/capellacollab/sessions/routes.py

```diff
@@ -159,10 +164,35 @@ def request_session(
         owner,
         rdp_password,
         db,
-        repository=body.repository,
+        repository=[],
```

Comment:
This parameter doesn’t accept a string, either the `create_database_and_guacamole_session` function has not been updated or the repository should be left unset.